modules/ximgproc/src/lsc.cpp

```diff
@@ -1165,7 +1165,7 @@ inline void SuperpixelLSCImpl::GetFeatureSpace()
     }
 
     // compute m_W normalization array
-    m_W = Mat( m_height, m_width, CV_32F );
+    m_W = Mat( m_height, m_width, CV_32F, 0.0f );
```

Comment:
BTW, be careful with 0 (0.0f is fine) - it may be interpreted as NULL for "data" pointer. `Mat::zeros(m_height, m_width, CV_32F)` should work here too.